react-native-libraries.json

```diff
@@ -16105,5 +16105,17 @@
     "web": true,
     "expoGo": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/maheshmuttintidev/react-native-scaler.git",
+    "npmPkg": "@shayrn/react-native-scaler",
+    "examples": [
+      "https://github.com/maheshmuttintidev/react-native-scaler?tab=readme-ov-file#-basic-usage",
+      "https://github.com/maheshmuttintidev/react-native-scaler/blob/main/EXAMPLE_USAGE.md"
+    ],
```

Comment:
Hey @maheshmuttintidev, thanks for adding a new package to the directory! 👍 

When it comes to examples, we only accept full apps or snacks:
* https://github.com/react-native-community/directory?tab=readme-ov-file#examples

Main link on the package name would lead to the package GitHub repository, so users will see the content in the main package README anyway.